library/core/functions.general.php

```diff
@@ -3110,6 +3110,32 @@ function safeImage($ImageUrl, $MinHeight = 0, $MinWidth = 0) {
     }
 }
 
```

Comment:
Sorry the comment has been made on the line above...

See: https://github.com/vanilla/vanilla/pull/5344/files/990a0b7d9a644e1ecf06392b8408f95e19df65c2#diff-604330a5cab4bf405103e84612897812R3113